concourse/pipelines/dev-image-build.yaml

```diff
@@ -366,3 +434,53 @@ jobs:
       publish_version: ((.:publish-version))
       wf: "debian/debian_10_worker.publish.json"
       environment: "prod"
+- name: publish-to-staging-rhel-7-metal-dev
+  plan:
+  - get: guest-test-infra
+  - get: compute-image-tools
+  - get: rhel-7-metal-dev-gcs
+    passed: [build-rhel-7-metal-dev]
+    trigger: false
+    params:
+      skip_download: "true"
+  - load_var: source-version
+    file: rhel-8-metal-dev-gcs/version
+  - task: get-credential
+    file: guest-test-infra/concourse/tasks/get-credential.yaml
+  - task: generate-version
+    file: guest-test-infra/concourse/tasks/generate-version.yaml
+  - load_var: publish-version
+    file: publish-version/version  # produced from generate-version task
+  - task: publish-rhel-7-metal-dev
+    file: guest-test-infra/concourse/tasks/daisy-publish-images.yaml
+    vars:
+      source_gcs_path: "gs://gce-image-archive/rhel"
+      source_version: v((.:source-version))
+      publish_version: ((.:publish-version))
+      wf: "linux_dev/rhel_7_metal_dev.publish.json"
+      environment: "staging"
```

Comment:
won't be passing environment with these workflows.

Reply:
remove this line